### PR TITLE
Bragi new params

### PR DESCRIPTION
**Pull request by remi-dupre**
Update Bragi API to latest changes, mostly poi dataset name (https://github.com/CanalTP/mimirsbrunn/pull/668).

### CODE REVIEW

File: idunn/geocoder/models/params.py [hunk 1]
```diff
@@ -30,54 +30,70 @@ class Type(str, Enum):
     Zone = "zone"
 
 
+class PlaceDocType(str, Enum):
+    # pylint: disable=invalid-name
+    Admin = "admin"
+    Street = "street"
+    Addr = "addr"
+    Poi = "poi"
+    Stop = "stop"
```
Comment (fatal69100) [inline idunn/geocoder/models/params.py:39]: I think Idunn should not be aware about `stop` type

**Review by fatal69100 — APPROVED** [review]

File: idunn/geocoder/models/params.py (hunk 1 above)
Comment (remi-dupre) [reply idunn/geocoder/models/params.py:39]: This makes sense :+1:

Comment (remi-dupre) [reply idunn/geocoder/models/params.py:39]: Do you think we should also remove `StopArea = "public_transport:stop_area"` from the enum `Type`?

Comment (fatal69100) [reply idunn/geocoder/models/params.py]: yes

**Pull request merged**